decode.go

```diff
@@ -16,11 +16,15 @@ type DecodeFeedback interface {
 	// is shorter than internal layer variables (HeaderLength, or the like) say it
 	// should be.  It sets packet.Metadata().Truncated.
 	SetTruncated()
+	// SetConsumed should be called when all payload has not been consumed
```

Comment:
Ok, the comment was updated.
This is used in our private layers which decode TCP protocols, see next comment.